tag_database

```diff
@@ -61,6 +61,10 @@ hasFlags:node
 head:array
 hexToRGB:utility,string,math,advanced
 hide:browser,css
+httpDelete:utility,url
+httpGet:utility,url
+httpPost:utility,url
+httpPut:utility,url
```

Comment:
Shouldn't all of these be under `browser`? Is `XMLHttpRequest` available in Node or other environments? (uncertain)

Reply:
Just added a browser tag to every one of them. Thanks for pointing out 👍